samples/matter/common/src/dfu/smp/dfu_over_smp.cpp

```diff
@@ -6,8 +6,9 @@
 
 #include "dfu_over_smp.h"
 
-#if !defined(CONFIG_MCUMGR_TRANSPORT_BT) || !defined(CONFIG_MCUMGR_GRP_IMG) || !defined(CONFIG_MCUMGR_GRP_OS)
-#error "DFUOverSMP requires MCUMGR module configs enabled"
+#if !defined(CONFIG_MCUMGR_TRANSPORT_BT) ||                                                                            \
+	(!defined(CONFIG_SUIT) && (!defined(CONFIG_MCUMGR_GRP_IMG) || !defined(CONFIG_MCUMGR_GRP_OS)))
+#error "DFUOverSMP requires MCUMGR module configs enabled and SUIT for the nRF54H20 platform"
```

Comment:
Could we split this into two separate error messages to be more informative? E.g.
```suggestion
#if !defined(CONFIG_MCUMGR_TRANSPORT_BT)
#error "DFUOverSMP requires MCUMGR module configs enabled"
#endif

#if (!defined(CONFIG_SUIT) && (!defined(CONFIG_MCUMGR_GRP_IMG) || !defined(CONFIG_MCUMGR_GRP_OS)))
#error  "DFUOverSMP requires SUIT configs enabled for the nRF54H20 platform"
#endif
```

Reply:
Sure.

Reply:
done.